static/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -526,25 +532,26 @@ class AddDashboardWidgetModal extends React.Component<Props, State> {
 
   async fetchMetricsTags() {
     const {api, organization, selection} = this.props;
-    const promise: Promise<MetricTag[]> = api.requestPromise(
-      `/organizations/${organization.slug}/metrics/tags/`,
-      {
-        query: {
-          project: !selection.projects.length ? undefined : selection.projects,
-        },
-      }
-    );
+    const projects = !selection.projects.length ? undefined : selection.projects;
+    const promise = fetchMetricsTags(api, organization.slug, projects);
+    const metricTags = await promise;
```

Comment:
I think you can probably one line it since there is no more try catch?
```suggestion
    const metricTags = await fetchMetricsTags(api, organization.slug, projects);
```